actions/keeper_helpers.go

```diff
@@ -39,7 +39,6 @@ func CreateKeeperJobs(chainlinkNodes []client.Chainlink, keeperRegistry contract
 			ContractAddress:          keeperRegistry.Address(),
 			FromAddress:              chainlinkNodeAddress,
 			MinIncomingConfirmations: 1,
-			ObservationSource:        client.ObservationSourceKeeperDefault(),
```

Comment:
We can probably rip out this function entirely.

Reply:
Thanks for pointing it out Adam! Just removed it now.